tests/TestEndpoints.js

```diff
@@ -40,6 +43,9 @@ export class TestEndpoints {
     return result;
   }
   async verifyVp(vp, options = {checks: []}) {
+    if (this.vpVerifier === null) {
+      return null;
```

Comment:
just throw here (unless in a before statement). That should result in the error report or at least the test runner telling them what is missing from their manifest. Please note 2 cases here: 1. they didn't add a vpVerifier section to their manifest 2. they added vpVerifiers but it had no matching tags and then case 3 they add a vpVerifier endpoint with some other missing prop such as tags. maybe we need a validator to run implementations on each endpoint.

Reply:
I will test these 3 use cases and report the current behavior

Reply:
The problem is that negative tests are expecting an error (`assert.rejects`) so returning an error actually counts the test as passed, hence why I'm returning a null value

Reply:
ahhh ok yeah I forgot that these test cases use a different assertion library so null is fine for now at least.